index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Scheherazade+New&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css2?family=Material+Icons" rel="stylesheet">
+    <title>African Devs</title>
+</head>
+
+<body>
+    <header>
+        <nav id="navbar">
+            <div class="nav-buttons">
+
+                <ul>
+                    
+                    <li>
+                        <a href="#projects">Projects</a>
+                    </li>
+                    <li>
+                        <a href="#skills">Skills</a>
+                    </li>
+                    <li>
+                        <a href="#contacts">Contacts</a>
+                    </li>
+    
+                </ul>
+            </div>
+
+            <div id="github_image">
+                
+                <a target="_blank" id="profile-link" href="https://github.com/matovu-farid/microverse_maxwell_farid/blob/85fcd846190f554111ebfb836d051ce16a43092b/index.html">
+                    <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/9/91/Octicons-mark-github.svg/1200px-Octicons-mark-github.svg.png" alt="Github">
+                    Github
+                </a>
+            </div>
+        </nav>
+    </header>
+    <section id="welcome-section">
+        <h1>
+            Welcome to my Portfolio
+        </h1>
+
+    </section>
+    <section id="projects">
+        <div>
+            <h1>Projects</h1>
```

Comment:
Wrap the heading in a header tag. This will help screen readers to know where the heading is on the browser.
```suggestion
<header>
            <h1>Projects</h1>
</header>
```

Reply:
- [ ] I'd suggest asking the author not to use more than one `h1` tag on the page🤔 so the page can be searched using SEO and be more semantic✌